src/backend/tests/conftest.py

```diff
@@ -361,7 +337,7 @@ async def test_user(client):
 
 
 @pytest.fixture
-def active_user(client):  # noqa: ARG001
```

Comment:
General thing: remove all changes that is not directly related to Gridgain support, please.

Reply:
confest file is linked with the tests folder when I ran the tests from current version of main branch it was causing errors from confest.py